pages/common/mongoimport.md

```diff
@@ -0,0 +1,24 @@
+# mongoimport
+
+> Imports content from a JSON, CSV, or TSV file into a MongoDB.
+> More information: <https://docs.mongodb.com/database-tools/mongoimport/>.
+
+- Import a JSON file into a specific collection:
+
+`mongoimport --file={{path/to/file}} --uri={{mongodb_uri}} --collection={{collection_name}}`
```

Comment:
```suggestion
`mongoimport --file={{path/to/file.json}} --uri={{mongodb_uri}} --collection={{collection_name}}`
```

Reply:
The extension is not mandatory and the user can use whatever custom extension that he wants.

Reply:
If it were mandatory, then we'd put it outside the `{{token_syntax}}`. Inside it's just a suggestion.

Reply:
I actually didn't know that we do that, but there are many pages using `{{file}}.ext`.

Reply:
Neither did I.  Though it makes sense, I don't _love_ the idea... 🤷🏻‍♂️

Reply:
I'm not really a fan either.
![image](https://user-images.githubusercontent.com/50295997/119190725-0526b300-ba7e-11eb-82c0-b779eb8b0e23.png)

Reply:
Yep, that's exactly why I'm not in favor of it.  The half-highligted page looks odd.